### ISSUE TITLE
Errors with babel 7.0.0-beta.46

### ISSUE THREAD

**Issue by throrin19** [egoist/poi#424]
**Do you want to request a *feature* or report a *bug*?**

Bug

**What is the current behavior?**

I update my packages in next version (poi, babel-preset-poi, babel/core, babel/polyfills) And now I have this error :

```
Module build failed: ReferenceError: Unknown option: .private. Check out http://babeljs.io/docs/usage/options/ for more information about options.
    at buildUnknownError (/project/Development/vue/vuejs-dashboard/node_modules/@babel/core/lib/config/validation/options.js:103:11)
    at /project/Development/vue/vuejs-dashboard/node_modules/@babel/core/lib/config/validation/options.js:89:57
    at Array.forEach (<anonymous>)
    at validate (/project/Development/vue/vuejs-dashboard/node_modules/@babel/core/lib/config/validation/options.js:67:21)
    at /project/Development/vue/vuejs-dashboard/node_modules/@babel/core/lib/config/config-chain.js:214:36
    at cachedFunction (/project/Development/vue/vuejs-dashboard/node_modules/@babel/core/lib/config/caching.js:42:17)
    at init (/project/Development/vue/vuejs-dashboard/node_modules/@babel/core/lib/config/config-chain.js:195:12)
    at /project/Development/vue/vuejs-dashboard/node_modules/@babel/core/lib/config/config-chain.js:279:17
    at mergeExtendsChain (/project/Development/vue/vuejs-dashboard/node_modules/@babel/core/lib/config/config-chain.js:342:19)
    at /project/Development/vue/vuejs-dashboard/node_modules/@babel/core/lib/config/config-chain.js:320:12
    at buildRootChain (/project/Development/vue/vuejs-dashboard/node_modules/@babel/core/lib/config/config-chain.js:90:27)
    at loadPrivatePartialConfig (/project/Development/vue/vuejs-dashboard/node_modules/@babel/core/lib/config/partial.js:51:53)
    at loadFullConfig (/project/Development/vue/vuejs-dashboard/node_modules/@babel/core/lib/config/full.js:43:37)
    at /project/Development/vue/vuejs-dashboard/node_modules/@babel/core/lib/transform.js:28:33
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
```

**Please mention other relevant information such as the browser version, Node.js version, Poi version and Operating System.**

```
{
  "@babel/polyfill": "7.0.0-beta.46",
  "@babel/core": "7.0.0-beta.46",
  "@poi/plugin-bundle-report": "10.0.0",
  "@poi/plugin-eslint": "10.0.1",
  "@poi/plugin-karma": "10.0.2",
  "@poi/plugin-offline": "10.0.0",
  "babel-eslint": "8.2.3",
  "babel-plugin-lodash": "3.3.2",
  "babel-plugin-lodash-modularize": "0.3.2",
  "babel-preset-poi": "10.0.1",
  "poi": "10.1.3"
}
```

**Comment by throrin19** [comment]
Ok I think I have understand the bug.

If we use the `babel` field inside package.json, actually, POI pass all `package.json` fields like option.

If we pass all this options in an `.babelrc` file, it works fine

**Comment by egoist** [comment]
Hmm seems we can't extend a `package.json` 😅

https://github.com/egoist/poi/blob/e451d119122d372afa35430c7cfb2b2f665b6159/packages/poi/lib/handleOptions.js#L24

**Comment by egoist** [comment]
Fixed in 10.1.4
